bazel/defaults.bzl

```diff
@@ -0,0 +1,21 @@
+load("@npm//@bazel/concatjs:index.bzl", _ts_library = "ts_library")
+
+def ts_library(name, testonly = False, deps = [], srcs = [], devmode_module = None, **kwargs):
+    current_pkg = native.package_name()
+
+    if current_pkg != "bazel" and not current_pkg.startswith("bazel/"):
+        fail("Cannot use `bazel/defaults.bzl` outside of the `bazel` directory. " +
+             "Use //tools:defaults.bzl instead")
+
+    _ts_library(
+        name = name,
+        devmode_module = devmode_module if devmode_module != None else "umd",
+        devmode_target = "es2020",
+        prodmode_module = "esnext",
+        prodmode_target = "es2020",
```

Comment:
This is a good idea. Thx.

Do you think might make sense that `//tools:defaults.bzl` uses this? so that we have a single place to maintain for this repo

Reply:
I think it would make sense, but I didn't do it because:
1. I didn't know the context of why `bazel` isn't allowed to import from `tools`.
2. The options are slightly different. E.g. `bazel` uses `devmode_module = "umd"` (same as `ts_library` default) whereas the one from `tools` uses `esnext`. Switching `bazel` to `esnext` ended up breaking some tests. The other difference is that `bazel` doesn't include `tslib` which there was also a test for.

Reply:
bazel folder is shipped as its own package so it cannot access anything outside.

I see on the second point. Feel free to merge this.

Reply:
It doesn't look like I have permission to merge it.